source/fundamentals/authentication/mechanisms.txt

```diff
@@ -208,9 +208,20 @@ The driver checks for your credentials in the following sources in order:
    .. tab:: Web Identity Token File
       :tabid: web-identity-token-file
 
-      You can use the OpenID Connect (OIDC) token obtained from a web identity provider to authenticate
-      to Amazon Elastic Kubernetes Service (EKS) or other services.
-      To use an OIDC token, create a file that contains your token, then 
+      You can use the OpenID Connect (OIDC) token obtained from a web identity
+      provider to authenticate to Amazon Elastic Kubernetes Service (EKS) or
+      other services.
+
+      To authenticate with your OIDC token you must first install
+      `@aws-sdk/credential-providers
+      <https://www.npmjs.com/package/@aws-sdk/credential-providers>`__. You can
+      install this dependency using the following ``npm`` command:
+
+      .. code-block:: bash
+
+         npm install @aws-sdk/credential-providers
+
+      Next, create a file that contains your OIDC token, then 
```

Comment:
[grammar]

I THINK this should be a period and a separate sentence, unless these are more closely connected than I realize.

```suggestion
      Next, create a file that contains your OIDC token. Then 
```